style.css

```diff
@@ -1,3 +1,9 @@
 body {
```

Comment:
- [ ] Kindly try to stick with the wireframe as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md#:~:text=Add%20CSS%20styles%20to%20the%20application%20to%20make%20it%20match%20this%20wireframe%3A)

| Your design | Wireframe | 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/176406643-a3ae52b3-79c7-4d9d-a0c8-b6574e736881.png) | ![image](https://user-images.githubusercontent.com/24703171/176219707-d0b736f8-915f-4da6-a881-568f47c0a894.png) |

You can use the styles of your choice (colors, fonts, backgrounds, etc.), but try to follow the general layout.